### PR TITLE
Revamp Containers > Amazon ECS page

### PR DESCRIPTION
**Pull request by cswatt**
### What does this PR do? What is the motivation?
Cleans up this page, updates instructions to align with current state of AWS console, removes outdated information.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/containers/amazon_ecs/_index.md [hunk 1]
```diff
@@ -27,49 +27,52 @@ algolia:
...
+To monitor your ECS containers and tasks, deploy the Datadog Agent as a container **once on each EC2 instance** in your ECS cluster. You can do this by creating a Task Definition for the Datadog Agent container and deploying it as a daemon service. Each Datadog Agent container then monitors the other containers on its respective EC2 instance.
 
-If you don't have a working EC2 Container Service cluster configured, review the [Getting Started section in the ECS documentation][4] to set up and configure a cluster. Once configured, follow the setup instructions below.
+The following instructions assume that you have configured an EC2 cluster. See the [Amazon ECS documentation for creating a cluster][4].
 
-1. [Create and add an ECS Task Definition](#create-an-ecs-task)
-2. [Schedule the Datadog Agent as a Daemon Service](#run-the-agent-as-a-daemon-service)
-3. **Optional** [Setup the additional Datadog Agent features](#setup-additional-agent-features)
+1. [Create and add an ECS task definition][27]
```
Comment (estherk15) [inline content/en/containers/amazon_ecs/_index.md:40]: Didn't know we could do this #TIL

File: content/en/containers/amazon_ecs/_index.md [hunk 2]
```diff
@@ -27,49 +27,52 @@ algolia:
...
-    2. If you are using Windows use [datadog-agent-ecs-win.json][22] 
+1. For Linux containers, download [datadog-agent-ecs.json][20].
+    - If you are using Amazon Linux 1 (AL1, formerly Amazon Linux AMI), use [datadog-agent-ecs1.json][21]
+    - If you are using Windows, use [datadog-agent-ecs-win.json][22] 
 
-2. Edit your base Task Definition file
-    1. Set `<YOUR_DATADOG_API_KEY>` with the [Datadog API key][14] for your account.
-    2. Set the `DD_SITE` environment variable to {{< region-param key="dd_site" code="true" >}} 
+   <div class="alert alert-info">
+   These files provide minimal configuration for core infrastructure monitoring. For more sample task definition files with various features enabled, see the <a href="#set-up-additioinal-agent-features">Set up additional Agent features</a> section on this page.
```
Comment (estherk15) [inline content/en/containers/amazon_ecs/_index.md:59]:
```suggestion
   These files provide minimal configuration for core infrastructure monitoring. For more sample task definition files with various features enabled, see the <a href="#set-up-additional-agent-features">Set up additional Agent features</a> section on this page.
```

File: content/en/containers/amazon_ecs/_index.md [hunk 3]
```diff
@@ -363,7 +411,7 @@ Need help? Contact [Datadog support][11].
 [1]: https://docs.datadoghq.com/integrations/faq/agent-5-amazon-ecs/
 [2]: https://docs.datadoghq.com/agent/docker/integrations/?tab=docker
 [3]: https://docs.datadoghq.com/integrations/ecs_fargate/
```
Comment (estherk15) [inline content/en/containers/amazon_ecs/_index.md:413]: I don't think these are referenced anywhere anymore? Also, [11] is not referenced.
```suggestion
```

**Review by estherk15 — APPROVED** [review]
Looks great! Some minor fixes for links, but approved!

File: content/en/containers/amazon_ecs/_index.md (hunk 2 above)
Comment (cswatt) [reply content/en/containers/amazon_ecs/_index.md:59]: omg this is a great catch

**Pull request merged**